.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: "gomod" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+    labels:
+    - "ok-to-test"
+    - "dependencies"
```

Comment:
should we add a label related to release note also

Reply:
good point, probably ?

Reply:
Or may be we can customize the commit message can put the release note block there with some standard note regarding dependency

Reply:
> Or may be we can customize the commit message can put the release note block there with some standard note regarding dependency

I am not sure about that, but yeah that could be nice. *Or*, update our create-draft-release task to parse dependabot commits/PR description.